### ISSUE TITLE
[Swift] height(forMapScale:) always returns -1

### ISSUE THREAD

**Issue by jmcazaux** [mousebird/WhirlyGlobe#879]
I have not been able to make this work. Initially I thought that it was because I was using it before the map was displayed, but after refactoring my code, no dice. Still returning -1.

**Comment by jmcazaux** [comment]
Forgot to mention, this is on branch master

**Comment by mousebird** [comment]
I'm looking at the code and it returns -1 if the renderer hasn't been set up with a proper framebuffer.

Can you try calling it after a few seconds and see if the problem remains?

**Comment by jmcazaux** [comment]
Yes, I have seen this as well but was not able to dig any further.

The issue is that this is part of the map initialisation process (I am adding various layers with different visibility and using heightForMapScale to set min and max visibility...

I tried moving the code from viewDidLoad to viewDidAppear, it did not change things.

Is there a callback (as we used to have on Android) that would be triggered when the map is fully initialized?

Having an empty map for a few seconds is not going to be an excellent user experience...
